test/assets/utils.js

```diff
@@ -125,6 +150,6 @@ module.exports = {
     logger: mainLogger,
     createRoom: createRoom,
     createSocket: createSocket,
-
+    getRoom: getRoomProject,
```

Comment:
There might be a better name for this. Currently `createRoom` and `getRoom` don't really tell what the difference between them is